src/utils/create_tmpfile.c

```
@@ -0,0 +1,42 @@
+	i = 0;
+	while (dirs[i])
+	{
+		path = ft_xstrjoin(dirs[i], target);
+		fd = open(path, O_WRONLY | O_CREAT | O_EXCL, S_IRUSR | S_IWUSR);
+		ft_strlcpy(tmp, path, TEMPLATE_LEN);
+		free(path);
+		if (fd != -1)
+			break ;
+		i++;
+	}
+	return (fd);
```

cppcheck

[warning] Uninitialized variable

(uninitvar)